### PR TITLE
feat(unify modules): merged widget with module 

### PR DESCRIPTION
**Pull request by AsifNawaz-cnic**

### CODE REVIEW

**Review by KaiSchwarz-cnic — CHANGES_REQUESTED** [review]
See my comments below.

plus: in keysystems.php, there's a line `require_once __DIR__ . '/vendor/autoload.php';` which represents the autload functionality of composer. Would you mind checking if loading of libraries also works if this line is not present? Feel free to discuss this with me for clarification.

File: modules/registrars/keysystems/keysystems.php [hunk 1]
```diff
@@ -1293,16 +1292,19 @@ function keysystems_GetZoneFeatures(array $params): ?object
  */
 function keysystems_getAccountDetails(): array
```
[the remark this answers is not in the document]
Comment (AsifNawaz-cnic) [reply modules/registrars/keysystems/keysystems.php:1293]: Thanks for notifying me, I have reverted my changes and applied Sebastian's changes.

**Comment by AsifNawaz-cnic** [comment]
> See my comments below.
> 
> plus: in keysystems.php, there's a line `require_once __DIR__ . '/vendor/autoload.php';` which represents the autload functionality of composer. Would you mind checking if loading of libraries also works if this line is not present? Feel free to discuss this with me for clarification.

Tried but did not worked, will need to create a separate task for this.

**Review by AsifNawaz-cnic — COMMENTED** [review]
Allowing dependabot to execute test.yml on pushes on all branches to lint the code and also resolved the conflicts.

**Comment by KaiSchwarz-cnic** [comment]
> Tried but did not work, will need to create a separate task for this.

Task created & added to backlog.

**Pull request closed without merging**